ember-bootstrap/addon/components/bs-dropdown/button.hbs

```diff
@@ -4,6 +4,10 @@
   {{! template-lint-disable no-capital-arguments no-args-paths }}
   @block={{this.args.block}}
   @onClick={{@onClick}}
+  @active={{@active}}
```

Comment:
`@active` argument does not seem to be covered by tests. Do you mind adding tests for it?